sourcecode/apis/contentauthor/stubs/cerpus/CreateTrait.stub

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Cerpus\Helper\Traits;
+
+trait CreateTrait
+{
+    /**
+     * @param mixed $attributes
```

Comment:
Array is the most common use, but `create()` also accepts a variable number of arguments.  This is used several places. No error is reported on these uses since they have DocBlocks. Even without DocBlocks, PHPStorm and PHPStan wouldn't have given warnings, since `create()` uses `func_get_args()`.